docs/conf.py

```diff
@@ -1,5 +1,15 @@
+import os
+import sys
+
 import sphinx_rtd_theme
 
+cwd = os.getcwd()
+project_root = os.path.dirname(cwd)
+sys.path.insert(0, project_root)
+
+import cryptoconditions     # noqa
```

Comment:
@r-marques  ok, done the change

as a side note, the disadvantage of reading the file is that its path is hardcoded, and if somehow the relative location of `conf.py` would change, with respect to `version.py`, we would need to update `conf.py` -- but that is really a minor detail at this point